Makefile

```diff
@@ -0,0 +1,31 @@
+#
+#  Copyright (c) 2021 International Business Machines
+#  All rights reserved.
+#
+#  SPDX-License-Identifier: LGPL-3.0-or-later
+#
+#  Authors: anita.shekar@ibm.com, sandy.kaur@ibm.com
+#
+
+MODULE := control
+CONFIG ?= ceph-nvmeof.conf
+
+setup: requirements.txt
+	pip3 install -r requirements.txt
+
+grpc:
+	@python3 -m grpc_tools.protoc \
+			--proto_path=./proto \
+			--python_out=./control/generated \
```

Comment:
I don't think this would work without a pre-created directory (e.g. on a fresh checkout).